python/orca/example/learn/pytorch/brainMRI/dataset.py

```diff
@@ -0,0 +1,82 @@
+import os
```

Comment:
This is not I have meant. I think you should close this PR, and open two new PRs:
1) The first one contain a `.py` file with exactly the same code as the original example - no need to add a jupyter notebook.
2) The second PR contains the necessary changes to make it run on Orca in a distributed fashion.

Reply:
I have two questions here:
1. First, the original example is a jupyter notebook; do I need to convert it into a `.py` file and upload it or just upload the notebook.
2. Secondly, how can I split them into two PR in one branch? 
Thanks.

Reply:
> I have two questions here:
> 
> 1. First, the original example is a jupyter notebook; do I need to convert it into a `.py` file and upload it or just upload the notebook.

convert it into a `.py` file 

> 2. Secondly, how can I split them into two PR in one branch?
>    Thanks.

merge the first PR first